docs/extensions/codecovergae-datacollector-issues.md

```diff
@@ -0,0 +1,17 @@
+# Syntax
```

Comment:
What about /enablecodecoverage or via settings?
Make it clear by adding one liner that this doc is related to the specified syntax